src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java

```diff
@@ -79,26 +77,26 @@
 import io.personium.test.categories.Unit;
 
 /**
- * TokenEndPointResource unit test classs.
+ * TokenEndPointResource unit test class.
  */
-@RunWith(PowerMockRunner.class)
```

Comment:
With PowerMockRunner, we can not somehow handle TransCellAccessToken.

Reply:
This test was using PowerMock excessively, so changed the whole test to minimize the use of Mocks